backend/src/services/secret-sync/render/render-sync-fns.ts

```diff
@@ -99,6 +110,18 @@ export const RenderSyncFns = {
   syncSecrets: async (secretSync: TRenderSyncWithCredentials, secretMap: TSecretMap) => {
     const renderSecrets = await getRenderEnvironmentSecrets(secretSync);
     for await (const key of Object.keys(secretMap)) {
+      // If value is empty skip it as render does not allow empty variables
+      if (secretMap[key].value === "") {
+        // According to @scott - we should be deleting them from thirdparty in this case
+        // if secret deletion is not disabled
+        if (!secretSync.syncOptions.disableSecretDeletion) {
+          await deleteEnvironmentSecret(secretSync, {
+            key
+          });
+        }
```

Comment:
is this deletion logic here still necessary given that we have the following below?

<img width="933" height="244" alt="image" src="https://github.com/user-attachments/assets/dab99431-6f03-4ce9-bcec-25ab4b4f2296" />

Reply:
then we probably wouldn't need this right?
<img width="682" height="157" alt="image" src="https://github.com/user-attachments/assets/8bd786f5-8977-4b13-b70b-fb668ffe419e" />